mix.exs

```diff
@@ -88,7 +88,7 @@ defmodule ElixirBoilerplate.Mixfile do
 
       # Test factories
       {:ex_machina, "~> 2.4", only: :test},
-      {:faker, "~> 0.12", only: :test},
+      {:faker, "~> 0.15", only: :test},
```

Comment:
This change doesn't impact the .lock because it was already generated with this version. 🤔